common/v2/features/Dashboard/constants.ts

```diff
@@ -7,7 +7,27 @@ import buyIcon from 'common/assets/images/icn-buy.svg';
 import swapIcon from 'common/assets/images/icn-swap.svg';
 import sendIcon from 'common/assets/images/icn-send.svg';
 import receiveIcon from 'common/assets/images/icn-receive.svg';
-import hardwareWalletIcon from 'common/assets/images/icn-hardware-wallet.svg';
+import ledgerIcon from 'common/assets/images/wallets/ledger.svg';
+import trezorIcon from 'common/assets/images/wallets/trezor.svg';
+
+const randomWallet = () => {
+  const hardwareWallets = [
+    {
+      icon: ledgerIcon,
+      title: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_TITLE'),
+      link: EXT_URLS.LEDGER_REFERRAL.path,
+      description: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_SUBTITLE', { $wallet: 'Ledger' })
+    },
+    {
+      icon: trezorIcon,
+      title: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_TITLE'),
+      link: EXT_URLS.TREZOR_REFERRAL.path,
+      description: translateRaw('DASHBOARD_ACTIONS_GET_WALLET_SUBTITLE', { $wallet: 'Trezor' })
+    }
+  ];
+
+  return hardwareWallets[Math.floor(Math.random() * hardwareWallets.length)];
```

Comment:
Please respect the separation of concerns and isolate the definition of an action vs selecting a random one from the list.